ci/test_python_common.sh

```diff
@@ -17,7 +17,7 @@ rapids-dependency-file-generator \
   --prepend-channel "${CPP_CHANNEL}" \
   --prepend-channel "${PYTHON_CHANNEL}" | tee env.yaml
 
-rapids-mamba-retry env create --yes -f env.yaml -n test
+rapids-mamba-retry env create --quiet --yes -f env.yaml -n test
```

Comment:
What is the goal of adding this `--quiet`?

For context... we'd recently tried to do that globally in CI images for RAPIDS, but found that it seemed to have the side effect of suppressing exception names from `mamba`, making `rapids-mamba-retry` ineffective:

https://github.com/rapidsai/ci-imgs/pull/220

`rapids-mamba-retry` / `rapids-conda-retry` work by string-matching on Python exceptions:

https://github.com/rapidsai/gha-tools/blob/0558ffce255e4e7da5d5312e79f35dd81e444144/tools/rapids-conda-retry#L82

So adding `--quiet` here might mean you're trading quieter logs for more need to manually retry failures from conda.

Reply:
Mostly added it here to reduce the number of lines I have to scroll past to see the output from the `pytest` command. It is somewhat annoying that the progress bars seem to take up many many many many lines :-/ but yeah, not really interested in negotiating with the rest of RAPIDS about this change :D

Reply:
> to reduce the number of lines I have to scroll past to see the output from the pytest command

I agree, it's annoying :/

That's why we'd attempted to make `quiet: true` the global setting for RAPIDS CI: https://github.com/rapidsai/ci-imgs/pull/217

But breaking the retry mechanism was just not worth it... I think having to scroll past some logs lines is better than having to manually re-run CI.

Anyway, the work to actually find the root cause of these excessive empty lines is still something we should do. Put up https://github.com/rapidsai/ci-imgs/issues/228 to track that.

Reply:
I assumed that it is something we'd have to fix in conda/mamba? Somehow making it aware of the fact that no human is watching so that it can either not output a progress bar or some such. At least it seems like there are CLI tools out there that somehow adjust the fancyness of their output. Alas, I have no idea how they do it :(

Reply:
Maybe, it might also be a side effect of the RAPIDS-specific wrappers we have around those tools. Put up one idea at https://github.com/rapidsai/ci-imgs/issues/228#issuecomment-2598757331

Anyway, that `ci-imgs` issue is a good tracking issue for this. Hopefully we can get more specific reproducible examples and make some progress there. I'll try to look into it when I can.